glide.yaml

```diff
@@ -72,3 +71,4 @@ import:
 - package: github.com/Shopify/sarama
   version: ^1.11.0
   repo: http://github.com/Shopify/sarama
+- package: github.com/rcrowley/go-metrics
```

Comment:
Can you give a short summary of: 
* the value this new dependency brings in (that we don't already have)
* why is that important to us
* why is this the right way, going forward, say compared to https://github.com/uber-go/tally

Unless there is a strong reason to pull in this dependency, I am not inclined to go down this road.
uber-go/tally is close to what we already have and provides the missing features on top of it.